staging/copy.sh

```diff
@@ -224,7 +224,7 @@ rm -rf "${CLIENT_REPO_TEMP}/staging"
 if [ "${FAIL_ON_CHANGES}" = true ]; then
   echo "running FAIL_ON_CHANGES"
   ret=0
-  if diff --ignore-matching-lines='^\s*\"Comment\"' -NauprB -I "GoVersion.*\|GodepVersion.*" "${CLIENT_REPO}" "${CLIENT_REPO_TEMP}"; then
+  if diff -NauprB  -I '^\s*\"Comment\"' -I "GoVersion.*\|GodepVersion.*" -x "base.go" "${CLIENT_REPO}" "${CLIENT_REPO_TEMP}"; then
```

Comment:
I tried to use the full path to be safe, but `diff -x` doesn't take a full path. i can do `find XXX | diff` but that looks like an overkill.